redhat/README.systemd

```diff
@@ -46,33 +51,38 @@ USER=add-the-user-id-you-want-to-use-here
 
 -----------------------------------------------
 
-* Passing extra arguments to the squeezeboxserver
+* Passing extra arguments to the Lyrion Music server
 
-The way the variable SQUEEZEBOX_ARGS is defined in /etc/sysconfig/squeezeboxserver
+The way the variable LYRION_ARGS is defined in /etc/sysconfig/lyrionmusicserver
 makes it unusable in the systemd unit file. If you have added parameters to
 this variable in the file and you want to keep them, then you must instead add
-them to the new variable SQUEEZEBOX_ADDITIONAL_ARGS. You can do this by adding 
-the following in the /etc/sysconfig/squeezeboxserver file (without the dashed
+them to the new variable LYRION_ADDITIONAL_ARGS. You can do this by adding 
+the following in the /etc/sysconfig/lyrionmusicserver file (without the dashed
 lines)
 
 ------------------------------------------------
 
-SQUEEZEBOX_ADDITIONAL_ARGS="--checkstrings --nomysqueezebox"
+LYRION_ADDITIONAL_ARGS="--checkstrings --nomysqueezebox"
```

Comment:
Please remove the reference to `--nomysqueezebox`, as that parameter no longer exists. It's been obsoleted.

Reply:
Actually... instead of the above example, use `--d_startup`, as that could actually be useful to some users, whereas `--checkstrings` could potentially be problematic (I believe it suffers from a memory leak).